gloom/src/main.cpp

```diff
@@ -9,6 +9,11 @@
 // Standard headers
 #include <cstdlib>
 
+// A callback which allows GLFW to report errors whenever they occur.
+static void glfwErrorCallback(int error, const char *description)
+{
+    fprintf(stderr, "GLFW returned an error:\n\t%s (%i)\n", description, error);
+}
```

Comment:
Add another blank space above and below the function. Also, remove punctuation after "_occur_" in the comment.